pkgs/tools/misc/turbo/default.nix

```diff
@@ -12,22 +12,23 @@
 , extra-cmake-modules
```

Comment:
```

nix-env failed:
error: undefined variable 'stdenv'

       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-0/pkgs/tools/misc/turbo/default.nix:41:33:

           40|
           41|     buildInputs = lib.optionals stdenv.isDarwin [ CoreServices ];
             |                                 ^
           42|
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/217476/commits/5eefe458715e624367ccd2287944a180436376c2